### ISSUE TITLE
Polygons Help

### ISSUE THREAD

**Issue by FlavianoMucedda** [google-map-react/google-map-react#613]
Hi!

I need to draw a polygon in an app on the google map.

I've already check all the issues (62 etc etc) and i really dont get it.

My lat and lng came from an api, i succeed without any problem to get them from the DB and it's been 4days i try to draw a damn polygon on the map without success (and i'm slightly gettin crazy...)

Can someone please help me? :)

**Comment by jabidof** [comment]
Try to build an SVG and add it within a react element having lat/lng as props.
Of course give a color to your svg.
Good luck

**Comment by itsmichaeldiego** [comment]
@FlavianoMucedda Could yo u provide a live example? I would love to help you but its not possible without something more tangible.

**Comment by bnanchen** [comment]
Hi!

I want to create a webpage containing a google map (using this library). The user has the possibility to upload a .svg file. The SVG is displayed on the map and the user can drag, rotate and scale it. Can library easily help me to accomplish the aim of the webpage?

I tried to use the solution given by @jabidof, but I can't come up with the correct implementation. I retrieve in a `prop` the SVG but then I don't understand how to build it within a react element having lat/lng as props. I tried this, but it doesn't work:
```
class SVGElement extends React.Component {
  render() {
    return {
      <div
      lat=46.466585
      lng=6.834234>
      {this.props.svg}<div/>
    }
  }
}
```

I'm new to react. Can someone help me?
